internal/platform/wrap_windows.go

```diff
@@ -0,0 +1,132 @@
+package platform
+
+import (
+	"io/fs"
+	"syscall"
+)
+
+// windowsWrappedFile deals with errno portability issues in Windows. This code
+// is likely to change as we complete WASI and GOOS=js.
+//
+// If we don't map to syscall.Errno, wasm will crash in odd way attempting the
+// same. This approach is an alternative to making our own fs.File public type.
+// We aren't doing that yet, as mapping problems are generally contained to
+// Windows. Hence, file is intentionally not exported.
+//
+// Note: Don't test for this type as it is wrapped when using sysfs.NewReadFS.
```

Comment:
trying to test for this type lost some time, hence `PathFile` workaround until we make our file type.